conda_env/cli/main_create.py

```diff
@@ -81,6 +88,8 @@ def execute(args, parser):
         common.error_and_exit(str(e), json=args.json)
 
     prefix = common.get_prefix(args, search=False)
+    if args.force and prefix != root_env_name and os.path.exists(prefix):
+        shutil.rmtree(prefix)
```

Comment:
Yes, please use `rm_rf`. `rmtree` is unreliable on Windows and `rm_rf` fixes many issues with it.